_plugins/author.rb

```
@@ -66,6 +65,13 @@ def with_pic(input)
         "<img class='img-circle team-img bio-clip' src='/assets/images/18f.png' alt='18F logo'>"
       end
     end
+    def lookup(input, args)
+      args = args.split(',')
+      dataset = args[0].strip
+      key = args[1].strip unless args[1].nil?
```

I'd `raise "property_name argument missing from 'lookup' filter"` or some such.